featuretools/entityset/entityset.py

```diff
@@ -800,6 +800,10 @@ def normalize_entity(self, base_entity_id, new_entity_id, index,
         # variable_types = base_entity.variable_types
         additional_variables = additional_variables or []
         copy_variables = copy_variables or []
+
+        if not (isinstance(additional_variables, list) and isinstance(copy_variables, list)):
```

Comment:
two comments

1. Let's split this into two checks. one for `additional_variables` and another for `copy_variables`.
2. Once we have two separate checks, let's make the error message more verbose e.g "additional_variables, must be a list, received type %s" % (type(additional_variables))"

Reply:
You are right! I will fix it within today.